pkg/asset/internal/templates.go

```diff
@@ -714,6 +714,7 @@ metadata:
   labels:
     k8s-app: kube-dns
     kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
In bootkube case, I don't think that's a bad thing (if someone wants to run the addonmanager)